pennylane/transforms/qcut.py

```diff
@@ -364,6 +366,158 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
     return tape
 
 
+def _get_measurements(
+    group: Sequence[Operator], measurements: Sequence[MeasurementProcess]
+) -> List[MeasurementProcess]:
+    """Pairs each observable in ``group`` with the fixed circuit ``measurements``.
+
+    Only a single fixed measurement of an expectation value is currently supported.
+
+    Args:
+        group (Sequence[Operator]): a collection of qubit-wise commuting observables
+        measurements (Sequence[MeasurementProcess]): fixed circuit measurements
+
+    Returns:
+        List[MeasurementProcess]: the expectation values of ``g @ obs``, where ``g`` is iterated
+        over ``group`` and ``obs`` is the observable composing the single fixed circuit measurement
+        in ``measurements``
+    """
+    n_measurements = len(measurements)
+    if n_measurements > 1:
+        raise ValueError(
+            "The circuit cutting workflow only supports circuits with a single output "
+            "measurement"
+        )
+    if n_measurements == 0:
+        return [expval(g) for g in group]
+
+    measurement = measurements[0]
+
+    if measurement.return_type is not Expectation:
+        raise ValueError(
+            "The circuit cutting workflow only supports circuits with expectation "
+            "value measurements"
+        )
+
+    obs = measurement.obs
+
+    return [expval(obs @ g) for g in group]
```

Comment:
Thanks for the addition and the explanation! This sounds like a sensible approach to me. Is this bug known and was there an issue raised for the bug? 

The above seems to work well with the current unit tests :ok_hand:

Reply:
> Is this bug known and was there an issue raised for the bug?

Trying to remember back - I wasn't sure exactly what was going on, and it could have been a bug in PL or in the prototype implementation. But yes I wouldn't say it's a well defined bug at this stage, and there isn't an issue. I'm sure we'll find it though when we get to the integration tests.